packages/maps/__tests__/Client.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { Maps as Client} from '../src/Client';
+
+describe('maps', () => {
+  it('can be easily created', () => {
+    const m = new Client('aUser', 'anApiKey');
+    expect(m).toBeTruthy();
+  });
+
+  it('requires credentials', () => {
+    expect(() => {
+      // tslint:disable-next-line: no-unused-expression
+      new Client('', '');
+    }).toThrow('Username is required');
```

Comment:
What about returning the expression? 😁
I guess that this would have the same result and allows us to remove the tslint override

```js
expect(() => new Client('', '')).toThrow('Username is required');
```

Reply:
Clearly better. Nice tip! 👍